Resources/Private/JavaScript/media-module/src/components/SideBarRight/Inspector/Property.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
 import { Label } from '@neos-project/react-ui-components';
 
-const Property: React.FC<{ label: React.ReactNode }> = ({ children, label }) => {
+const Property: React.FC<{ label: React.ReactNode; className?: string }> = ({ children, label, className }) => {
     return (
-        <div className="propertyGroup">
+        <div className={className ? 'propertyGroup' + ' ' + className : 'propertyGroup'}>
```

Comment:
Instead of providing a classname I would prefer a boolean property that adds the class based on some semantic meaning.

Reply:
i refactored the property component to render differently if the cild component is a checkbox. This information is passed via a prop. I think this is a little bit better dann an additional class.